frontend/package-lock.json

```diff
@@ -1612,224 +1612,208 @@
       "license": "MIT"
     },
     "node_modules/@rollup/rollup-android-arm-eabi": {
-      "version": "4.21.1",
-      "resolved": "https://registry.npmjs.org/@rollup/rollup-android-arm-eabi/-/rollup-android-arm-eabi-4.21.1.tgz",
-      "integrity": "sha512-2thheikVEuU7ZxFXubPDOtspKn1x0yqaYQwvALVtEcvFhMifPADBrgRPyHV0TF3b+9BgvgjgagVyvA/UqPZHmg==",
+      "version": "4.22.4",
+      "resolved": "https://registry.npmjs.org/@rollup/rollup-android-arm-eabi/-/rollup-android-arm-eabi-4.22.4.tgz",
+      "integrity": "sha512-Fxamp4aEZnfPOcGA8KSNEohV8hX7zVHOemC8jVBoBUHu5zpJK/Eu3uJwt6BMgy9fkvzxDaurgj96F/NiLukF2w==",
       "cpu": [
         "arm"
       ],
       "dev": true,
-      "license": "MIT",
```

Comment:
The `license` key is removed in `package.lock.json` for the updated packages. Based on this [bug report](https://github.com/npm/cli/issues/7384) it looks like there's some history to licenses in that file. And there should be a fix in npm v10.8.0. (Haven't checked yet which npm version dependabot uses.)

So it seems we shouldn't care that much about the license in `package.lock.json`? @praseodym What do you think?